### PR TITLE
Initial portal config

### PR DESCRIPTION
**Pull request by dhhuynh2**
Initial portal config for common-ios. There seems to be no documentation, so I didn't create an mkdocs.yml file.

### CODE REVIEW

File: portal.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+
+   
+apiVersion: backstage.io/v1alpha1
```
Comment (ryanmrodriguez) [inline portal.yml]:
```suggestion
apiVersion: backstage.io/v1alpha1
```

☺️

File: portal.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+kind: Component
+metadata:
+  name: common-ios
+  title: Common iOS
+  description: Common additions to the UIKit and Foundation frameworks used by Power Home iOS projects
+spec:
+  type: library
+  owner: rebel-alliance
+  system: nitro-ios
+  lifecycle: production
```
Comment (ryanmrodriguez) [inline portal.yml]: \n

🧐

File: portal.yml (hunk 1 above)
Comment (dhhuynh2) [reply portal.yml]: https://github.com/powerhome/common-ios/pull/1/commits/04bb2a61f46344d22dce194a950ca40ba159b485

File: portal.yml (hunk 2 above)
Comment (dhhuynh2) [reply portal.yml]: https://github.com/powerhome/common-ios/pull/1/commits/04bb2a61f46344d22dce194a950ca40ba159b485

**Review by ryanmrodriguez — APPROVED** [review]

**Pull request merged**